StRoot/StFttClusterMaker/StFttClusterMaker.h

```diff
@@ -0,0 +1,65 @@
+/***************************************************************************
+ *
+ * $Id: StFttClusterMaker.h,v 1.3 2019/02/19 20:32:05 fseck Exp $
```

Comment:
$Id$ is leftover from CVS days.   Can be removed.

Reply:
Addressed: 3686a6eea3b300c95180bdcee0ad698b5763a88c